qa/rpc-tests/prioritisetransaction.py

```diff
@@ -28,23 +28,28 @@ def setup_nodes(self):
             '-debug=zrpcunsafe',
         ]
         return start_nodes(self.num_nodes, self.options.tmpdir, extra_args=[
+            args + ['-txunpaidactionlimit=51', '-blockunpaidactionlimit=50'],
+            args + ['-txunpaidactionlimit=51', '-blockunpaidactionlimit=32'],
             args + ['-txunpaidactionlimit=51'],
-            args + ['-txunpaidactionlimit=51', '-blockunpaidactionlimit=25'],
-            args + ['-txunpaidactionlimit=51'],
-            args + ['-txunpaidactionlimit=25'],
+            args + ['-txunpaidactionlimit=32'],
```

Comment:
It's 1:40am and I'm too tired to understand and review the changes to this test. I did not review this file.

Reply:
Yes this is extending the coverage of the test as well as fixing it. I *could* split out the extensions into a different commit, but they're pretty entangled tbh. The reason for that is that the required number of blocks means that the test has to span regtest halvings, which complicates everything. I could maybe add more comments.